src/DIRAC/FrameworkSystem/Client/ComponentInstaller.py

```diff
@@ -2107,7 +2107,7 @@ def installPortal(self):
 #
 exec 2>&1
 #
-exec python %(DIRAC)s/WebAppDIRAC/scripts/dirac-webapp-run.py -p < /dev/null
+exec python %(DIRAC)s/WebAppDIRAC/scripts/dirac_webapp_run.py -p < /dev/null
```

Comment:
Has this script been renamed?
https://github.com/DIRACGrid/WebAppDIRAC/tree/integration/scripts

Reply:
Indeed no, but it should. @TaykYoku

Reply:
Should it just be this instead as the absolute path also won't work with Python 3?
```suggestion
exec dirac-webapp-run -p < /dev/null
```